modules/web-mercator/src/web-mercator-utils.ts

```diff
@@ -331,7 +331,9 @@ export function getProjectionParameters(options: {
   const horizonDistance = cameraToSeaLevelDistance * 10;
 
   // Calculate z value of the farthest fragment that should be rendered.
-  const farZ = Math.min(furthestDistance * farZMultiplier, horizonDistance);
+  // Add a bit extra to avoid precision problems when a fragment's distance is exactly `furthestDistance`
+  // 3.x backward compatibility: should produce mapbox equivalent when farZMultiplier=1.01
+  const farZ = Math.min(furthestDistance * 1.01, horizonDistance) * (farZMultiplier / 1.01);
```

Comment:
I would expect farZmultiplier=1.0 to match mapbox. After all this code was designed to match mapbox. Why Would it be 1.01 ?